components/brave_wallet/browser/block_tracker.cc

```diff
@@ -14,12 +15,27 @@ BlockTracker::BlockTracker(JsonRpcService* json_rpc_service)
   DCHECK(json_rpc_service_);
 }
 
+BlockTracker::~BlockTracker() = default;
+
+void BlockTracker::Stop(const std::string& chain_id) {
+  if (base::Contains(timers_, chain_id)) {
+    timers_[chain_id]->Stop();
+    timers_.erase(chain_id);
+  }
+}
+
 void BlockTracker::Stop() {
-  timer_.Stop();
+  for (const auto& timer : timers_) {
+    timer.second->Stop();
+  }
+  timers_.clear();
```

Comment:
Deleting timer is enough, no need to Stop()

Reply:
fixed in fedd8a6e93192ead54bde1ad8a99896f9b15f5f7